### ISSUE TITLE
Re-using the browser cache at the initial measurement trial

### ISSUE THREAD

**Issue by gogo9th** [sitespeedio/browsertime#1233]
Hi,

I am having a trouble in re-using the browser cache when using browsertime. In particular, I need to measure the pageload time and speed index after for a logged-in webpage. Therefore, I need to run browsertime, log in, close it, and re-start browsertime. But when I re-start browsertime, it doesn't seem to use the previous session's browser cache. For some reason, it seems to force not using the browser cache at the initial measurement trial. I wonder how we can make it re-use the browser cache from the 1st trial.

**Comment by soulgalore** [comment]
Hi @gogo9th the cache is always cleared when you re-open the browser in the current setup. The way you can do it is to run multiple script (start with a login, then measure what you want, then ...). Would that help you?

**Comment by gogo9th** [comment]
Hi @soulgalore, I first tried to login to Gmail after launching the browser for https://gmail.com, but it doesn't work: 

[http://attach.mail.daum.net/bigfile/v1/urls/d/W8LMjqQwMI4rdINIVAYoxB-tgDE/XDUehjxRoEtvFMwd6jsrHQ)](Screenshot)

And I tried to inject "console.log('hello')" by the following:

`nodejs ../node_modules/browsertime/bin/browsertime --pageCompleteCheckPollTimeout 10000 -injectJs 'console.log("Hello");' --chrome.args 'auto-open-devtools-for-tabs'  https://gmail.com`

. . . but the console doesn't seem to print hello.

Lastly, I wonder if you know how to inject Javascript at the point of onLoad instead at the beginning of the document loading (injectJs).

Thanks!
